### DISCUSSION TITLE
Syncing across browers

### DISCUSSION THREAD

**Discussion by robbyking** [ankit/stylebot#609] [Ideas]
Are plans to allow syncing across browsers/profiles? I use multiple computers, and would like to be able to sync my settings between them. Tab Mix Plus is a great example of this functionality in a Chrome Plugin. 

Also, I'd be happy to contribute to the project!

**Comment by ankit** [comment]
great idea @robbyking. I have something already in the works at https://github.com/ankit/stylebot/tree/sync :) (sync via Google Drive)
